src/openlcb/TractionTrain.cxx

```diff
@@ -429,24 +446,32 @@ struct TrainService::Impl
             }
             uint8_t cmd = payload()[0];
             bool flip_speed = false;
-            if (cmd == TractionDefs::REQ_SET_SPEED) {
-                if (flags & TractionDefs::CNSTFLAGS_REVERSE) {
+            if (cmd == TractionDefs::REQ_SET_SPEED)
+            {
```

Comment:
I appreciate cleanups in the code, but please do not mix a feature implementation with random cleanups. It makes it take unnecessarily long to review the change.
Two changes, one whitespace-only and one meaningful takes much shorter to review total.

Reply:
Sorry let me pull this PR and clean it up in two PRs instead.

Reply:
actually that does not matter anymore because the effort was spent -- just for the future please.

Reply:
I see a few cleanups I missed as well in here. I'll resubmit this PR once the comments are addressed.